cmd/root.go

```diff
@@ -109,4 +123,5 @@ func init() {
 	rootCmd.Flags().StringP("policy-server-url", "u", "", "Full URL to the PolicyServers, for example https://localhost:3000. Audit scanner will query the needed HTTP path. Useful for out-of-cluster debugging")
 	rootCmd.Flags().VarP(&level, "loglevel", "l", fmt.Sprintf("level of the logs. Supported values are: %v", logconfig.SupportedValues))
 	rootCmd.Flags().BoolVarP(&printJSON, "print", "p", false, "print result of scan in JSON to stdout")
+	rootCmd.Flags().StringP("skipped-ns", "s", "", "Comma separated list of namespace names to be skipped from scan")
```

Comment:
I'm not super thrilled of getting the long list of skipped ns as a CSV in a flag. But I don't want to use a config file, and an env var would also consist of a CSV.

Reply:
nitpicking, maybe we should use "ignore" instead of "skip"

Reply:
done in https://github.com/kubewarden/audit-scanner/pull/53/commits/0b6f6f72ceae73ad7226377810b85d887723d731.